### ISSUE TITLE
Update the token scopes needed

### ISSUE THREAD

**Issue by froi** [github/ghec-audit-log-cli#19]
**Describe the bug**

This really isn't a bug, more like an issue with our docs. The README indicates that the [token must have the `admin:org` scope](https://github.com/github/ghec-audit-log-cli#secret-values) to be able to read the audit log, yet the output I got mentions that more are required.

Some of the scopes needed that were mentioned in the output were"

- `read:user`
- `public_repo`
- `user:email`

**To Reproduce**
1. Create a PAT that only has the `admin:org` scope
2. [Run the cli](https://github.com/github/ghec-audit-log-cli#running-the-cli) or [install the CLI](https://github.com/github/ghec-audit-log-cli#installing-as-cli)
3. Pass the token and org CLI arguments
4. See error output

**Expected behavior**

I think the CLI is behaving as expected. I think the README should be updated and maybe rearranged to indicate that the `admin:org` scope is the minimum scope needed.

**Screenshots**

Not really screenshot but here's the copied CLI output: [output.log](https://github.com/github/ghec-audit-log-cli/files/6259249/output.log)

**Additional context**

I was testing this on an org that is part of a GHEC account.

**Comment by adamkadzban** [comment]
I ran into a similar issue. I'm not sure if it's exactly the same since this issue is pretty old, but I specified the v3 api and the documented scopes were enough.

**Comment by froi** [comment]
@droidpl Hey there 👋🏼 

I've been going over "my" outstanding issues and found that this is still open. There seems to be other users facing the same issue.

Has this been addressed? Can we close this out? Thanks!

**Comment by droidpl** [comment]
@adamkadzban Not sure if you managed to solve the issue, for some reason missed this notification. Let me know, otherwise happy to help.

@froi I think we haven't picked this up indeed, but if you have some time happy to review a PR from an old friend 😄  Otherwise I'll put this to my TODO list

**Comment by adamkadzban** [comment]
@droidpl I was able to get it working. The issue is that the documentation lists the scopes that you need when using the v3 api, but the newer/default v4 api requires additional scopes. It really should just be a documentation update, I think?

**Comment by droidpl** [comment]
yep, to me tat looks like the only change required. What are the additional scopes? @adamkadzban will you submit the change? Happy to review a PR 😸
